python/ray/_private/parameter.py

```diff
@@ -156,7 +157,9 @@ def __init__(
         include_dashboard: Optional[bool] = None,
         dashboard_host: Optional[str] = ray_constants.DEFAULT_DASHBOARD_IP,
         dashboard_port: Optional[bool] = ray_constants.DEFAULT_DASHBOARD_PORT,
-        dashboard_agent_listen_port: Optional[int] = 0,
+        dashboard_agent_listen_port: Optional[
```

Comment:
Maybe not optional anymore? What's the semantic of optional?

Reply:
@simon-mo @shrekris-anyscale Do you know about optional?

Reply:
I'm not sure about why this is `Optional`.